amaabca/ama_layout
# ADO 17518: Support Rails 6

### PR Review
[AB#72417](https://dev.azure.com/AMA-Ent/ccb94584-03a5-426b-9211-75a5115f75dc/_workitems/edit/72417)

See also: https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/17518

* use Draper instead of copying what Draper does. Draper is updated
  for Rails 6
* fixes related to Rails 6 support changes


### How To Test
- install this gem on any rails app we have (e.g gatekeeper) and run random UI test, this should not have an impact or errors
